workflow/cron/operator.go

```diff
@@ -227,17 +239,28 @@ func (woc *cronWfOperationCtx) shouldOutstandingWorkflowsBeRun() (bool, error) {
 }
 
 func (woc *cronWfOperationCtx) reconcileActiveWfs(workflows []v1alpha1.Workflow) error {
+	updated := false
```

Comment:
could this func race with others in the same context?

Reply:
No, it is key-lock protected